go/core.rst

```diff
@@ -527,7 +527,7 @@ Attributes
 | :param:`msan`              | :type:`string`              | :value:`auto`                         |
 +----------------------------+-----------------------------+---------------------------------------+
 | This is one of the `mode attributes`_ that controls whether to instrument                        |
-| code for memory santization. It may be :value:`on`, :value:`off`, or                             |
+| code for memory sanitization. It may be :value:`on`, :value:`off`, or                             |
```

Comment:
Please make sure the right edge is aligned. The table won't render without this.

Reply:
Fixed